src/chunk_constraint.c

```diff
@@ -546,13 +573,26 @@ chunk_constraint_create_on_chunk(Chunk *chunk, Oid constraint_oid)
 							chunk->fd.hypertable_id);
 }
 
+typedef struct HypertableConstraintOperation
+{
+	const char *hypertable_constraint_name;
+} HypertableConstraintOperation;
+
 typedef struct DeleteConstraintInfo
 {
+	const char *hypertable_constraint_name;
 	int32		chunk_id;
```

Comment:
`chunk_id` doesn't seem to be used in this struct, so unnecessary?

Reply:
Yup, fixed.